ddtrace/contrib/internal/selenium/patch.py

```diff
@@ -108,26 +135,50 @@ def _handle_enter(self):
         if webdriver_instance is None:
             return
 
-        is_rum_active = webdriver_instance.execute_script(RUM_STOP_SESSION_SCRIPT)
-        time.sleep(FLUSH_SLEEP_MS / 1000)
+        is_rum_active = webdriver_instance.execute_script(_RUM_STOP_SESSION_SCRIPT)
+        time.sleep(config.selenium.flush_sleep_ms / 1000)
 
         if is_rum_active:
-            root_span.set_tag("test.is_rum_active", True)
+            root_span.set_tag("test.is_rum_active", "true")
 
         webdriver_instance.delete_cookie("datadog-ci-visibility-test-execution-id")
 
 
-def get_version():
+def get_version() -> str:
+    import selenium
+
     return selenium.__version__
 
 
-def patch():
-    SeleniumGetWrappingContext(WebDriver.get).wrap()
-    SeleniumQuitWrappingContext(WebDriver.quit).wrap()
-    SeleniumQuitWrappingContext(WebDriver.close).wrap()
+def patch() -> None:
+    import selenium
+
+    if getattr(selenium, "_datadog_patch", False):
+        return
+
+    @when_imported("selenium.webdriver.remote.webdriver")
+    def _(m):
+        global _WRAPPING_CONTEXTS_STORES
+        for wrapper_tup in [
+            (SeleniumGetWrappingContext, m.WebDriver.get),
+            (SeleniumQuitWrappingContext, m.WebDriver.quit),
+            (SeleniumQuitWrappingContext, m.WebDriver.close),
+        ]:
+            wrapper = wrapper_tup[0](wrapper_tup[1])
```

Comment:
Isn't this the same as:
```suggestion
        for wrapper in [
            SeleniumGetWrappingContext(m.WebDriver.get),
            SeleniumQuitWrappingContext(m.WebDriver.quit),
            SeleniumQuitWrappingContext(m.WebDriver.close),
        ]:
```

Reply:
Ah yeah, hadn't even thought of that. Will update.